### DISCUSSION TITLE
Mass insert - ON DUPLICATE KEY UPDATE

### DISCUSSION THREAD

**Discussion by mareksloser** [nextras/orm#536] [Q&A]
Hello,

at first i would like thank you for this ORM.

Could you help me solve my problem? I would like avoid duplicate value in DB. I have M:M relationship. Could help me how i can avoid hudge query number for check if value exist in db? I found "ON DUPLICATE KEY UPDATE" but i dont if i can use in this ORM.

Thank you

"nextras/orm": "^4.0",

**Comment by hrach** [comment]
This basically is not much feasible using Orm. If the amount is huge, the best would be do this manually with Nextras\Dbal. What do you think about this approach?
